showcase/src/app/getting-started/getting-started-component/starter-structure.ts

```diff
@@ -0,0 +1,125 @@
+export const starterStructure: string = `|
++---config
+|   |
+|   |   webpack-custom-config.dev.json  # webpack configuration for the development environment
+|
++---src
+|   |
+|   +---app
+|   |   |
+|   |   +---foo                         # smart component folder
+|   |   |   |                           # create a folder for each component
+|   |   |   |   foo.component.css       # styles for this smart component
+|   |   |   |   foo.component.html      # template for this smart component
+|   |   |   |   foo.component.spec.ts   # jasmine test  for this smart component
```

Comment:
In the PrettyPrint component, the text on the right in this code block is not well aligned, could you align them?

![unaligned-pretty-print-1](https://user-images.githubusercontent.com/5698318/48055157-38024780-e1af-11e8-9f2d-b3fa7a2280b2.PNG)